packages/runtime/src/account.ts

```diff
@@ -85,6 +85,28 @@ export class StoreAccount implements StoreAccountI {
     });
   }
 
+  /**
+   * Fetches global state value for key present in account
+   * returns undefined otherwise
+   * @param appId: current application id
+   * @param key: key to fetch value of from local state
+   */
+  getGlobalState (appId: number, key: Uint8Array | string): StackElem | undefined {
```

Comment:
Just to confirm, we haven't removed this function from `runtime` right? `runtime.getGlobalState` will call `account.getGlobalState`. Right?

Reply:
yes